src/main/java/com/cloudbees/jenkins/GitHubCommitNotifier.java

```diff
@@ -129,7 +129,7 @@ private void updateCommitStatus(@Nonnull AbstractBuild<?, ?> build, @Nonnull Bui
                 }
 
                 listener.getLogger().println(Messages.GitHubCommitNotifier_SettingCommitStatus(repository.getUrl() + "/commit/" + sha1));
-                repository.createCommitStatus(ObjectId.toString(sha1), state, build.getAbsoluteUrl(), msg);
+                repository.createCommitStatus(ObjectId.toString(sha1), state, build.getAbsoluteUrl(), msg, "jenkins");
```

Comment:
@oleg-nenashev do you really want to have another one configuration in UI? Correct job name will have correct commit status.